### ISSUE TITLE
Add multiple validations to a single property

### ISSUE THREAD

**Issue by alexmartinezm** [reactiveui/ReactiveUI.Validation#3]
**Is your feature request related to a problem? Please describe.**
Actually, if you try to add two validations to the same property, eg `Password`, in the ValidationContext you would do it like this:

```csharp
[…]
this.ValidationRule(
     vm => vm.Password,
     password => !string.IsNullOrEmpty(password),
     "Password is required.");

this.ValidationRule(
     vm => vm.Password,
     password => password.Length > 5,
     "Password minimum length is 5.");
[…]
```

And from the View you would bind to the `Password` property like this:

```csharp
[…]
this.BindValidation(ViewModel, vm => vm.Password, view => view.PasswordErrorMessage.Text)
    .DisposeWith(disposables);
[…]
```

The problem by doing this is that from the `ValidationBindings`, used by the `ViewForExtensions`, it looks up only for the first or default validation rule (see `ResolveFor` methods in `ValidationContextExtensions`).

**Describe the solution you'd like**
Add support for multiple validations in the same property since it's supported in the ValidationContext side.

**Describe suggestions on how to achieve the feature**
Basically, if more than one property matches the query in the `ResolveFor` method then unless the Formatter has been provided, it should throw a custom exception like `MultipleValidationNotSupportedException`.

On the other side, we should provide Mixins to give support for multiple validation scenarios.

**Comment by worldbeater** [comment]
> We should provide Mixins to give support for multiple validation scenarios.

I'd prefer this option over throwing exceptions ✨
